android-components/components/service/contile/src/test/resources/contile/contile-hasExclusions.json

```diff
@@ -0,0 +1,9 @@
+{
+  "tiles": [
```

Comment:
I think we should add some sample tiles here since we are expecting the exclusion to filter filter the tiles from this list.

Reply:
Like I said above, what we actually wanna filter is in default top sites. This is why I add another test in [DefaultTopSitesStorageTest.kt](https://github.com/mozilla-mobile/firefox-android/pull/898/files#diff-6fab8584751338c2c0e0e23f17cd92baaa44bd33d92dc6fe6f83486b2a4ad1ba)